haskell-src/Concordium/Types/ProtocolVersion.hs

```diff
@@ -471,9 +479,7 @@ supportsV1Contracts :: SProtocolVersion pv -> Bool
 supportsV1Contracts SP1 = False
 supportsV1Contracts SP2 = False
 supportsV1Contracts SP3 = False
-supportsV1Contracts SP4 = True
-supportsV1Contracts SP5 = True
-supportsV1Contracts SP6 = True
+supportsV1Contracts _ = True
```

Comment:
Please don't use wildcard here. I think these configurations should be very explicit to avoid accidents.